validphys2/src/validphys/covmats.py

```diff
@@ -472,133 +472,10 @@ def groups_corrmat(groups_covmat):
     return mat
 
 
-@check_data_cuts_match_theorycovmat
-def dataset_inputs_covmat(
-        data: DataGroupSpec,
-        fitthcovmat,
-        t0set:(PDF, type(None)) = None,
-        use_weights_in_covmat: bool = True,
-        norm_threshold=None):
-    """Like `covmat` except for a group of datasets"""
-    if not use_weights_in_covmat:
-        data = data.to_unweighted()
-
-    loaded_data = data.load()
-
-    if t0set:
-        #Copy data to avoid chaos
-        loaded_data = type(loaded_data)(loaded_data)
-        log.debug("Setting T0 predictions for %s" % data)
-        loaded_data.SetT0(t0set.load_t0())
-
-    covmat = loaded_data.get_covmat()
-
-    if fitthcovmat:
-        loaded_thcov = fitthcovmat.load()
-        ds_names = loaded_thcov.index.get_level_values(1)
-        indices = np.in1d(ds_names, [ds.name for ds in data.datasets]).nonzero()[0]
-        covmat += loaded_thcov.iloc[indices, indices].values
-    if norm_threshold is not None:
-        covmat = regularize_covmat(
-            covmat,
-            norm_threshold=norm_threshold
-        )
-    return covmat
-
-
-@check_dataset_cuts_match_theorycovmat
-def covmat(
```

Comment:
Where has all this code and documentation gone? How are  the `norm_threshold` and `use_weights_in_covmat` arguments handled?

Reply:
The weights are used in the lower level functions https://github.com/NNPDF/nnpdf/blob/eeae19b15c960b4f8f974c3ac2ba2af36ed845bc/validphys2/src/validphys/covmats.py#L35